### PR TITLE
Techs styles

### PR DESCRIPTION
**Pull request by tammaroivan**
## Summary
-Tech card styles

## Screenshot
image

## Trello Card

https://trello.com/c/4XXqjFCi/22-tecnologia

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -11,5 +11,15 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1> 
+
```
Comment (FrankIglesias) [inline maquetado/index.html]: check intentation and delete extra empty line

Comment (FrankIglesias) [reply maquetado/index.html]: use two spaces

File: maquetado/index.html [hunk 2]
```diff
@@ -11,5 +11,15 @@
...
+	
+	<div class="cards">
+		<div class="tech">
+			<img src="assets/react_icon.svg" alt="">
+			<p>ReactJS</p>
+			<p><strong>Release: 2013</strong></p>
+			<p class="description">React is declarative, component-base JavaScript library for building user interfaces.</p>
+		</div>
+	</div>
+
```
Comment (FrankIglesias) [inline maquetado/index.html]: delete empty line

File: maquetado/scss/styles.scss [hunk 3]
```diff
@@ -1,3 +1,15 @@
+$green: #BED23C;
+
+* {
+	font-family: 'Roboto', sans-serif;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: same

File: maquetado/scss/styles.scss [hunk 4]
```diff
@@ -1,3 +1,15 @@
+$green: #BED23C;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: move this to a `colors.scss` file

File: maquetado/scss/styles.scss [hunk 5]
```diff
@@ -1,3 +1,15 @@
...
+
+* {
+	font-family: 'Roboto', sans-serif;
+}
+
 body {
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: you can replace this with a margin of the section container instead

File: maquetado/scss/tech.scss [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+	border-radius: 5px;
+	flex-direction: column;
+	justify-content: center;
+	text-align: center;
+	padding: 15px;
+	position: relative;
+	width: 200px;
+	height: 300px;
+
+	img {
```
Comment (FrankIglesias) [inline maquetado/scss/tech.scss]: dont use tags in scss

File: maquetado/scss/tech.scss [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+	p {
+		margin: 5px 0;
+	}
+
+	.description{
+		margin: 10px 0;
+	}
+}
+
+.tech::after {
```
Comment (FrankIglesias) [inline maquetado/scss/tech.scss]: nest this inside tech.. like 
```
.tech {
....
  &::after {
...
  }
}
```

File: maquetado/scss/tech.scss [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+	position: relative;
+	width: 200px;
+	height: 300px;
+
+	img {
+		width: 125px;
+		margin: 5px 0;
+	}
+
+	p {
```
Comment (FrankIglesias) [inline maquetado/scss/tech.scss]: same

**Review by FrankIglesias — CHANGES_REQUESTED** [review]
check the indentation configuration of atom

File: maquetado/index.html (hunk 1 above)
Comment (tammaroivan) [reply maquetado/index.html]: done

File: maquetado/index.html (hunk 2 above)
Comment (tammaroivan) [reply maquetado/index.html]: done

File: maquetado/scss/styles.scss (hunk 3 above)
Comment (tammaroivan) [reply maquetado/scss/styles.scss]: done

File: maquetado/scss/styles.scss (hunk 4 above)
Comment (tammaroivan) [reply maquetado/scss/styles.scss]: done

File: maquetado/scss/tech.scss (hunk 6 above)
Comment (tammaroivan) [reply maquetado/scss/tech.scss]: done

File: maquetado/scss/tech.scss (hunk 7 above)
Comment (tammaroivan) [reply maquetado/scss/tech.scss]: done

File: maquetado/scss/tech.scss (hunk 8 above)
Comment (tammaroivan) [reply maquetado/scss/tech.scss]: done

File: maquetado/scss/styles.scss [hunk 9]
```diff
@@ -1,3 +1,8 @@
-body {
-	padding: 100px;
-}
\ No newline at end of file
+* {
+  font-family: 'Roboto', sans-serif;
+}
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: add empty line between classes

File: maquetado/scss/tech.scss [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+  height: 300px;
+
+  .logo {
+    width: 125px;
+    margin: 5px 0;
+  }
+  .text {
+    margin: 5px 0;
+  }
+  .description{
```
Comment (FrankIglesias) [inline maquetado/scss/tech.scss]: same

File: maquetado/scss/tech.scss [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+    margin: 10px 0;
+  }
+  &::after {
```
Comment (FrankIglesias) [inline maquetado/scss/tech.scss]: same

File: maquetado/scss/tech.scss [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    border: 2px solid $green;
+    border-radius: 5px;
+    left: -13px;
+    top: -13px;
+    z-index: -1;
+  }
+
```
Comment (FrankIglesias) [inline maquetado/scss/tech.scss]: delete this one

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**